### PR TITLE
Added Null Check to Components

### PR DESCRIPTION
**Pull request by sachinsom93**
### Fix Issue: #42 
Added NullCheck of props to:
    1. ButtonWithHeading
    2. Contact
    3. Footer
    4. GitterRoomItem
    5. Guildlines
    6. MediumFeed
    7. pageNotFound

### CODE REVIEW

File: src/components/ButtonWithHeading/index.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ export const ButtonWithHeading = ({ heading, buttonText, onClick = () => {}}) =>
       <Container>
         <Row>
           <Col> 
-            <h1>{heading}</h1>
+            <h1>{!heading || (heading)}</h1>
```
Comment (subodhk01) [inline src/components/ButtonWithHeading/index.js:12]: Don't you think a cleaner way to null check would be { heading && (heading) }

File: src/components/PageNotFound/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const PageNotFound = ({ image, mainText, subText }) => {
     <div className="page-not-found-component">
       <div className="inner-container">
         <img src={withPrefix(image)} alt='404'/>
-          <h1>{mainText}</h1>
+          {!image || (<h1>{mainText}</h1>)}
```
Comment (subodhk01) [inline src/components/PageNotFound/index.js:11]: why is image prop used to null check mainText?

**Review by subodhk01 — CHANGES_REQUESTED** [review]

File: src/components/ButtonWithHeading/index.js (hunk 1 above)
Comment (sachinsom93) [reply src/components/ButtonWithHeading/index.js:12]: OK, will do the changes, thanks.

File: src/components/PageNotFound/index.js (hunk 2 above)
Comment (sachinsom93) [reply src/components/PageNotFound/index.js:11]: My bad, mainText should be used to null check mainText.

**Comment by sachinsom93** [comment]
Hi @subodhk01, I've done the changes, do I need to change anything else ? Thanks for your feedback.

File: src/components/ButtonWithHeading/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ export const ButtonWithHeading = ({ heading, buttonText, onClick = () => {}}) =>
       <Container>
         <Row>
           <Col> 
-            <h1>{!heading || (heading)}</h1>
+            <h1>{ heading && (heading)}</h1>
```
Comment (subodhk01) [inline src/components/ButtonWithHeading/index.js:12]: Just saw the codebase, ! is used everywhere. So you can revert this commit to maintain code consistency

**Review by subodhk01 — CHANGES_REQUESTED** [review]

File: src/components/ButtonWithHeading/index.js (hunk 3 above)
Comment (sachinsom93) [reply src/components/ButtonWithHeading/index.js:12]: Which commit ?

Comment (sachinsom93) [reply src/components/ButtonWithHeading/index.js:12]: So, do I need to use ! again for null check ?

Comment (subodhk01) [reply src/components/ButtonWithHeading/index.js:12]: Yeah

Comment (sachinsom93) [reply src/components/ButtonWithHeading/index.js]: Done.

**Comment by Grumpyyash** [comment]
Null checks were implemented in further PRs. Thanks @sachinsom93 for your contribution :)

**Pull request closed without merging**